modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -12,15 +12,17 @@ module Mobile
   module V0
     class ClaimsAndAppealsController < ApplicationController
       include IgnoreNotFound
-      before_action(only: %i[index get_claim request_decision]) do
+      before_action(only: %i[get_appeal]) { authorize :appeals, :access? }
+
+      before_action(only: %i[get_claim request_decision]) do
         if Flipper.enabled?(:mobile_lighthouse_claims, @current_user)
           authorize :lighthouse, :access?
         else
           authorize :evss, :access?
         end
       end
 
-      before_action(except: %i[index get_claim request_decision]) do
+      before_action(except: %i[index get_appeal get_claim request_decision]) do
```

Comment:
Does appeals require evss access? If so, should the `appeals_acces?` method also check for evss access?

Reply:
appeals doesn't require evss access since it goes through caseflow. So only `:appeals, :access? ` is needed

Reply:
Sorry, I misread this as an "only" and thought this check was being run. I think he changed it to be an "only" per the discussion he and I had here: https://github.com/department-of-veterans-affairs/vets-api/pull/13284#discussion_r1266768440

I wonder if that change was reverted by a merge. It might be worth changing it back to `only`.

Reply:
looks like that pr is still open. Do you think i should still make the change here or leave it for his pr?

Reply:
Didn't realize it was still open. AFAICT from looking at his PR, it looks like that EVSS check is going away entirely. I think you just need to make sure that any changes you make are correct and let him deal with the rest. Since you added logic for `get_appeal`, you can re-add `get_appeal` to the `except` list and then your specs will be right and Jason can handle the rest.

Reply:
ok its been readded